packages/block-library/src/tag-cloud/style.scss

```diff
@@ -18,4 +18,18 @@
 		margin-left: 5px;
 		text-decoration: none;
 	}
+
+	&.is-style-pill a {
+		border: 1px solid var(--wp--preset--color--foreground);
+		border-radius: 5px;
+		font-size: var(--wp--preset--font-size--normal);
+		margin-bottom: 10px;
```

Comment:
We might want to use some factor of block gap here.

Reply:
Can it be a percentage so that if font-size is changed it works? Same for padding.

Reply:
I don't think a percentage will work, but I updated it to use `ch` units which does. See the update screenshots above...

Reply:
Sweet!